scripts/extract_dts_includes.py

```diff
@@ -397,14 +400,21 @@ def extract_single(node_address, yaml, prop, key, prefix, defs, def_label):
 
   return
 
-def extract_property(yaml, node_address, y_key, y_val, names, prefix, defs):
+def extract_property(node_compat, yaml, node_address, y_key, y_val, names, prefix, defs, label_override):
 
   node = reduced[node_address]
-  def_label = convert_string_to_label(get_compat(node)).upper()
-  def_label += '_' + node_address.split('@')[-1].upper()
+
+  if yaml[node_compat].get('base_label') is not None:
+    def_label = yaml[node_compat].get('base_label')
+  else:
+    def_label = convert_string_to_label(node_compat.upper())
+    def_label += '_' + node_address.split('@')[-1].upper()
+
+  if label_override != None:
```

Comment:
PEP008: "Comparisons to singletons like None should always be done with is or is not , never the equality operators."

https://www.python.org/dev/peps/pep-0008/